index.html

```diff
@@ -10,42 +10,189 @@
+                <li><button class="js" type="button">JavaScript</button></li>
+            </ul>
+            <div class="pjbutton">
+                <a class="actbutton" href="project5.html">See this project</a>
+                <img class="narrow" src="./images/narrow.png" alt="ico_narrow">
+            </div>
+        </div>
+    </section>
+    <section class="project">
+        <img src="./images/pcprogect5.png" alt="Project6">
+        <div class="box">
+            <p>Project name goes here</p>
+            <ul>
+                <li><button class="html" type="button">HTML/CSS</button></li>
+                <li><button class="ruby" type="button">Ruby on Rails</button></li>
+                <li><button class="js" type="button">JavaScript</button></li>
+            </ul>
+            <div class="pjbutton">
+                <a class="actbutton" href="project6.html">See this project</a>
+                <img class="narrow" src="./images/narrow.png" alt="ico_narrow">
+            </div>
+        </div>
+    </section>
+    <section class="project">
+        <div class="box1">
+            <h2>About</h2>
+            <p class="phrase">
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. 
+                xercitationem iure cum impedit velit, accusantium quod voluptatem 
+                deleniti doloremque. Debitis et placeat quae optio, dicta dignissimos 
+                reiciendis facilis vel quclassem explicabo.
+            </p>
+            <div class="pjbutton">
+                <a class="actbutton1" href="project6.html">Get my CV</a>
+                <img class="narrow0" src="./images/download.png" alt="ico_download">
+            </div>
+        </div>
+    </section>
+    <section class="project">
+        <div id="square1">
+            <h3>Languages</h3>
+            <ul class="chart">
+                <li>JavaScript (ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+            <img id="narrow1" src="./images/sharp.png" alt="ico_sharp">
+        </div>
+    </section>
+    <section class="project">
+        <div id="square2">
+            <h3>Frameworks</h3>
+            <ul class="chart">
+                <li>Botstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <li>Selenium</li>
+            </ul>
+            <img id="narrow2" src="./images/framework.png" alt="ico_framework">
+        </div>
+    </section>
+    <section class="project">
+        <div id="square3">
+            <h3>Skills</h3>
+            <ul class="chart">
+                <li>Codekit</li>
+                <li>Codepen</li>
+                <li>GitHub</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+            </ul>
+            <img id="narrow3" src="./images/light.png" alt="ico_light">
+        </div>
+    </section>
```

Comment:
- [ ] The same thing for the `about` section, let's wrap all the sections in one section and change the `section` tag to `div` tag for each section. see the following structure:
```html
<section>
  <div class="project">
        <div class="box1">
            <h2>About</h2>
            ...
  </div>
  <div class="project">
        <div id="square1">
            <h3>Languages</h3>
            <ul class="chart">
               ...
        </div>
  </div>
  ...
</section>
```

Reply:
section tag well-implemented 👍🏽